src/components/HomePage.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+/**
+ * Required Props:
+ * navBar {React Component} – Component representing the page nav bar
+ */
+function HomePage(props) {
```

Comment:
What's the point of this component? Couldn't `<Homepage navBar={navBar} />` always be replaced with just `<Navbar />`